build.sbt

```diff
@@ -38,8 +38,8 @@ lazy val scalaHttp = crossProject.in(file("."))
     ),
     pomIncludeRepository := { _ => false },
 
-    libraryDependencies += "com.lihaoyi" %%% "utest" % "0.4.4",
-    libraryDependencies += "io.monix" %%% "monix" % "2.0.6",
+    libraryDependencies += "com.lihaoyi" %%% "utest" % "0.4.5" % Test,
```

Comment:
it's the safer version http://www.scala-sbt.org/1.0/docs/Library-Dependencies.html#Per-configuration+dependencies . I think `utest` only need in Test configuration so it's stricter to only add `utest` to the project `Test`.